website/profile/views.py

```diff
@@ -524,6 +531,10 @@ def impute_names(**kwargs):
     return auth_utils.impute_names(name)
 
 
+def update_osf_help_mails_subscription(user, subscribe):
+    user.osf_mailing_lists['Open Science Framework Help'] = subscribe
```

Comment:
Please use `settings.OSF_HELP_LIST` instead of writing out the list name. Please do a quick search and replace all instances of "Open Science Framework Help" with its settings variable `OSF_HELP_LIST`